### PR TITLE
Readded quickwit projects in CI

### PR DESCRIPTION
**Pull request by fulmicoton**

### CODE REVIEW

File: quickwit/Cargo.toml [hunk 1]
```diff
@@ -36,9 +36,38 @@ members = [
...
   "quickwit-indexing",
-  "quickwit-metastore"
+  "quickwit-ingest",
+  "quickwit-integration-tests",
+  "quickwit-jaeger",
+  "quickwit-janitor",
+  "quickwit-macros",
+  "quickwit-macros/impl",
+  "quickwit-metastore",
+  # Disabling metastore-utils from the quicwit projects to ease build/deps.
```
Comment (trinity-1686a) [inline quickwit/Cargo.toml:60]: typo
```suggestion
  # Disabling metastore-utils from the quickwit projects to ease build/deps.
```

File: quickwit/Cargo.toml [hunk 2]
```diff
@@ -36,9 +36,38 @@ members = [
...
+  "quickwit-datetime",
+  "quickwit-directories",
+  "quickwit-doc-mapper",
   "quickwit-indexing",
-  "quickwit-metastore"
+  "quickwit-ingest",
+  "quickwit-integration-tests",
+  "quickwit-jaeger",
+  "quickwit-janitor",
+  "quickwit-macros",
```
Comment (trinity-1686a) [inline quickwit/Cargo.toml:57]: I guess we don't want `quickwit-lambda` here, but maybe put it commented to be explicit?

**Review by trinity-1686a — APPROVED** [review]

File: quickwit/Cargo.toml (hunk 2 above)
Comment (rdettai) [reply quickwit/Cargo.toml:57]: Yes, Lambda tests are not run in CI. @fmassot can we enable them now?

Comment (fmassot) [reply quickwit/Cargo.toml:57]: I would avoid that for now.

**Pull request merged**